lessons/operator/03/README.md

```diff
@@ -1,370 +1,212 @@
+   The output should include a value for `SENSU_VERSION` (i.e. a release version such as `6.2.7`).
 
-![The configuration navigation](img/03-10-configuration-navigation.png)
+   **Example Output:**
 
-_**Figure 10:** monitoring configuration views are accessible via the navigation sidebar._
+   ```shell
+   SENSU_VERSION=6.2.7
+   SENSU_BUILD=4449
+   SENSU_BACKEND_HOST=127.0.0.1
+   SENSU_NAMESPACE=default
+   SENSU_USER=sensu
+   SENSU_PASSWORD=sensu
+   SENSU_BACKEND_URL=ws://127.0.0.1:8081
+   SENSU_API_URL=http://127.0.0.1:8080
+   ```
 
-![The check list view](img/03-11-check-list-view.png)
+1. **Download and Install `sensuctl`.**
 
-_**Figure 11:** the check list view.
-A similar view is available for Filters, Mutators, and Handlers._
+   Download the platform-specific archive from Sensu's official release location. Unpack the archive into a standard location.
 
-![The check detail view](img/03-12-check-detail-view.png)
+   **Mac:**
 
-_**Figure 12:** the check detail view.
-A similar view is available for Filters, Mutators, and Handlers._
+   ```shell
+   curl -LO "https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/${SENSU_VERSION}/sensu-go_${SENSU_VERSION}_darwin_amd64.tar.gz"
+   sudo tar -xzf "sensu-go_${SENSU_VERSION}_darwin_amd64.tar.gz" -C /usr/local/bin/
+   ```
 
-![The check edit view](img/03-13-check-edit-view.png)
+   **Linux:**
 
-_**Figure 13:** the check edit view.
-A similar editor is available for Filters, Mutators, and Handlers._
+   ```shell
+   curl -LO "https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/${SENSU_VERSION}/sensu-go_${SENSU_VERSION}_linux_amd64.tar.gz" 
+   tar -xzf "sensu-go_${SENSU_VERSION}_linux_amd64.tar.gz" -C /usr/local/bin/ 
+   ```
+ 
+   **Windows (Powershell):**
 
-## EXERCISE 1: log in to the Sensu web app
+   ```powershell
+   Invoke-WebRequest `
+     -Uri "https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/${Env:SENSU_VERSION}/sensu-go_${Env:SENSU_VERSION}_windows_amd64.zip" `
+     -OutFile "${Env:UserProfile}\sensu-go_${Env:SENSU_VERSION}_windows_amd64.zip"
+   Expand-Archive `
+     -LiteralPath "${Env:UserProfile}\sensu-go_${Env:SENSU_VERSION}_windows_amd64.zip" `
+     -DestinationPath "${Env:UserProfile}\Sensu\bin"
+   ${Env:Path} += ";${Env:UserProfile}\Sensu\bin"
+   ```
 
-1. **Log in to the Sensu web app.**
+   > **NOTE:** On Unix-like systems `/usr/local/bin` is the [standard location for binaries shared between multiple users][fhs_usr_local_docs]. 
+   > On macOS, the default permissions for `/usr/local/bin` require the used of `sudo`. 
+   > However, any location that is on your `$PATH` will work (i.e. `~/bin`).
+   > On Windows systems we create a new location specificly for Sensu, then add it to the path.
 
-   Please visit http://127.0.0.1:3000 and login with the default workshop admin username (`sensu`) and password (`sensu`).
+## Configuration Options
```

Comment:
Good point, but I think I'll skip on this one. I think the context is sufficient to disambiguate that.

Also, one of the linguistic nits here is that any time I refer to `sensuctl` I want to do so w/ proper case and code formatting (e.g. in backticks). So that title would look like:

**`sensuctl` Configuration Options**

Which does not look good, because the first letter should be capitalized and using the code-symbol formatting in the heading font sizing just looks awkward, especially when it's the first word. 

I made sure to phrase things so that no sentences start with "<blah blah>. Sensuctl <blah blah>" or "<blah blah> . `sensuctl` <blah blah>" because of this.